bitnami/spark/templates/statefulset-master.yaml

```
@@ -344,3 +344,6 @@ spec:
             secretName: {{ template ".Values.initScriptsSecret" . }}
             defaultMode: 0755
         {{- end }}
+  {{- if .Values.worker.extraVolumeClaimTemplates }}
+  volumeClaimTemplates: {{- include "common.tplvalues.render" (dict "value" .Values.worker.extraVolumeClaimTemplates "context" $) | nindent 8 }}
+  {{- end }}
```

I think you forgot to modify this, the value used should be `.Values.master.xxxx`:

```suggestion
  {{- if .Values.master.extraVolumeClaimTemplates }}
  volumeClaimTemplates: {{- include "common.tplvalues.render" (dict "value" .Values.master.extraVolumeClaimTemplates "context" $) | nindent 8 }}
  {{- end }}
```

---

Thank you for review. When I sow an email and changed it without looking at Git Hub, so now I could not Sign off and commit suggestion that you provided. Sorry.
However, I changed it as you suggested.